tests/pkg_elk/main.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2021 Cesanta Software Limited
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
```

Comment:
GPLv2 ;-).

Reply:
(and since it is not under LGPLv2.1, maybe don't reference the LICENSE file here)